File: C++/Data_Structure/Linear_Search/Linear_Search.cpp
#include "Linear_Search.hpp"
#include <iostream>

int LinearSearch(int target, std::vector<int> targetList) {
  int index;
  int location;
  while (index <= targetList.size() && target != targetList[index]) {
    index += 1;
  }
  if (index <= targetList.size()) {
    location = index;
  } else {
    return -1;
  }
  std::cout << location;
  return location;
};

File: Python/Discrete Math/Algo/Linear Search.py
def Linear_Search(X,a):
    i=1
    while(i<=len(a) and X!=a[i]):
        i+=1
    if(i<=len(a)):
        location = i
    else:
        location = 0
    return location

print(Linear_Search(3,[1,2,3,4,5,6]))